HermaFx.Rebus/RebusExtensions.cs

```diff
@@ -52,5 +53,17 @@ public static void Reply<TResponse>(this IBus bus, Action<TResponse> customizer)
 
 			bus.Reply(message);
 		}
+
+		public static void ReplyToOriginator<TResponse>(this IBus bus, IMessageContext messageContext, TResponse response)
+		{
+			Guard.IsNotNull(bus, nameof(bus));
+			Guard.IsNotNull(messageContext, nameof(messageContext));
+			Guard.IsNotNull(response, nameof(response));
+
+			var correlationKey = Headers.CorrelationId;
+
+			bus.AttachHeader(response, correlationKey, messageContext.Headers[correlationKey]?.ToString());
```

Comment:
CorrelationKey should be passed as argument here.. caller should not need to know we will be trying to access correlationKey at messageContext... that seems a little bit dumb.